mis_builder/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'MIS Builder',
-    'version': '9.0.2.0.2',
+    'version': '10.0.2.0.2',
```

Comment:
Well, you will have the same conflicts cherry-picking/merging commits from previous branch, and it's not the regular norm, but you are the maintainer of this module, so you can decide in this.